Makefile

```diff
@@ -15,14 +15,32 @@ docker-build:
 	docker build -t cjchung4849/scenario-manager:dev -f docker/scenario.Dockerfile .
 	docker push cjchung4849/scenario-manager:dev
 # Compute
-#	docker build -t phudtran/merak-compute:dev -f docker/compute.Dockerfile .
-#	docker build -t phudtran/merak-compute-vm-worker:dev -f docker/compute-vm-worker.Dockerfile .
-#	docker push phudtran/merak-compute:dev
-#	docker push phudtran/merak-compute-vm-worker:dev
+	make proto
+	docker build -t phudtran/merak-compute:dev -f docker/compute.Dockerfile .
+	docker build -t phudtran/merak-compute-vm-worker:dev -f docker/compute-vm-worker.Dockerfile .
+	docker push phudtran/merak-compute:dev
+	docker push phudtran/merak-compute-vm-worker:dev
+# Agent
+	docker build -t phudtran/merak-agent:dev -f docker/agent.Dockerfile .
+	docker push phudtran/merak-agent:dev
```

Comment:
@phudtran Please include all other components into this Makefile and also add MIT license. Thanks.

Reply:
I can add the license.
But this docker directive will need to be changed once we have a team docker hub account.
It won't work if I add it for other team members since we cannot login into multiple accounts on the same machine